### ISSUE TITLE
rfswarm becomes unresponsive when clicking the CSV Report button 

### ISSUE THREAD

**Issue by ryanrosello-og** [damies13/rfswarm#128]
**Describe the bug**

**To Reproduce**
1. Run simple load test
2. Wait for completion
3. Click the CSV Report button 

**Expected behavior**
CSV file generated

**Screenshots**
![2023-02-14_11-23-38](https://user-images.githubusercontent.com/50574915/218616713-24842eaa-a9b5-4c02-8de1-68394bd609d0.jpg)

```
Robot Framework Swarm: Manager
        Version 1.0.0
        Configuration File:  C:\.wrk\.autotests\xxxautomation\.automation\lib\site-packages\rfswarm_manager\RFSwarmManager.ini
C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py:839: UserWarning: Tight layout not applied. The left and right margins cannot be made large enough to accommodate all axes decorations.
  func(*args)
C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py:839: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations.
  func(*args)
Test Started:    1676338492 [ 2023-02-14 11:34:52 ]
Run Stopped: 1676338496 [ 2023-02-14 11:34:56 ]
Test Started:    1676338509 [ 2023-02-14 11:35:09 ]
Exception in thread Thread-1459 (<lambda>):
Traceback (most recent call last):
  File "C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "C:\.wrk\.autotests\xxxautomation\.automation\lib\site-packages\rfswarm_manager\rfswarm.py", line 4491, in <lambda>
    tsm = threading.Thread(target=lambda: self.gs_updatesecmetrics(grphWindow))
  File "C:\.wrk\.autotests\xxxautomation\.automation\lib\site-packages\rfswarm_manager\rfswarm.py", line 4636, in gs_updatesecmetrics
    grphWindow.fmeMSettings.omSM.set_menu(*grphWindow.fmeMSettings.SMetrics)
  File "C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\tkinter\ttk.py", line 1643, in set_menu
    menu.delete(0, 'end')
  File "C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py", line 3389, in delete
    self.deletecommand(c)
  File "C:\Users\ROSELLOR7C\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py", line 673, in deletecommand
    self.tk.deletecommand(name)
_tkinter.TclError: can't delete Tcl command
Test Completed:  1676338582 [ 2023-02-14 11:37:24 ]
```

**Desktop (please complete the following information):**
- Windows 11
- Python 3.10.9

**Comment by damies13** [comment]
Hi Ryan,

Thanks for the bug report, I'll try to reproduce it and figure out what's happening.

Dave.

**Comment by damies13** [comment]
Hi Ryan,

Quick question, did you have a graph window open at the graph settings?

The reason i ask is these lines in the stack trace are from the graph settings:
```
  File "C:\.wrk\.autotests\xxxautomation\.automation\lib\site-packages\rfswarm_manager\rfswarm.py", line 4491, in <lambda>
    tsm = threading.Thread(target=lambda: self.gs_updatesecmetrics(grphWindow))
  File "C:\.wrk\.autotests\xxxautomation\.automation\lib\site-packages\rfswarm_manager\rfswarm.py", line 4636, in gs_updatesecmetrics
    grphWindow.fmeMSettings.omSM.set_menu(*grphWindow.fmeMSettings.SMetrics)
```

Could you try again without any graph windows open (just once for troubleshooting)?

Clicking the CSV Report button should have created a dialogue like this 
![image](https://raw.githubusercontent.com/damies13/rfswarm/v1.1.0/Doc/Images/Linux-v0.5.0_Run_Report_prompt.png)

But even if it didn't it could be worth checking the results folder to see if a .csv or .txt file was created? that will help me know where it's failing.

Dave.

**Comment by ryanrosello-og** [comment]
hey @damies13 

I tried it again this time without opening any graphs:

https://user-images.githubusercontent.com/50574915/218652867-d6752ff7-8d14-4e6d-a8aa-e2e9f31ce74b.mp4

In the results folder, I don't see any .txt or csv files generated.
![2023-02-14_16-00-21](https://user-images.githubusercontent.com/50574915/218652908-9859ed07-8715-427a-8179-3c610fe0b995.jpg)

**Comment by ryanrosello-og** [comment]
I suspect this could be the problem:

```
[Scenario]
uploadmode = err
scriptcount = 1
graphlist = 

[1]
robots = 2
delay = 0
rampup = 10
run = 60
test = Create article
script = ..\..\..\..\tests\performance\create_article.robot
robotoptions = TEST_ENV:dev
```
^ I did not have `-v` in the robotoptions

**Comment by damies13** [comment]
> hey @damies13
> 
> I tried it again this time without opening any graphs:
> 
> In the results folder, I don't see any .txt or csv files generated. 

I noticed in that video there was no measurements on the run screen (perhaps related to #129 ?) 

I really don't remember but there might be some logic not to create an empty csv file.

Lets resolve  #129 and get you some data to put in the csv file.

**Comment by damies13** [comment]
After figuring out issue #129 I tried generating a CSV file:

Almost imminently after clicking the button I g
![Screenshot from 2023-02-26 14-53-57](https://user-images.githubusercontent.com/15232871/221393090-4b233cbf-7d5a-4b11-8671-b4f632d0966f.png)
ot the dialogue:

and opening the file it looks like this:
![Screenshot from 2023-02-26 14-54-16](https://user-images.githubusercontent.com/15232871/221393096-9e127150-e071-42f5-b5ce-cf813c2975d7.png)

**Comment by damies13** [comment]
Also created files for the agent history and the raw data

![Screenshot from 2023-02-26 14-58-39](https://user-images.githubusercontent.com/15232871/221393209-35a48fc2-b388-4165-bbfa-06b2bb9476fb.png)

![Screenshot from 2023-02-26 14-59-04](https://user-images.githubusercontent.com/15232871/221393213-4eb86319-0130-4b07-acd9-a4c69783dea4.png)

**Comment by damies13** [comment]
Fix for this bug unit tested and merged into v1.0.1 release branch
